### DISCUSSION TITLE
Downside to excluding/deleting transfers from one asset account to another

### DISCUSSION THREAD

**Discussion by asmartin** [firefly-iii/firefly-iii#8152] [Firefly III] [answered]
I am mostly interested in using firefly-iii for transaction categorization (e.g. to see total spending per category). Unfortunately with the way my accounts are setup I don't think I can create automatic rules to correctly link things like credit card payments (e.g. linking a withdrawal from a checking account to the credit card account) as described in the docs. What are the downsides to just deleting these "internal" transfer transactions between accounts that are all imported into firefly-iii? Yes, it means the account balances will not be accurate, but since I am only concerned about categorizing transactions, this isn't a feature I am concerned with. Is there anything else I should consider before mass deleting all of these transfers?

Thank you!

**Comment by JC5** [comment, marked as answer]
No real downsides, no. It should be safe :)

**Reply by asmartin** [reply]
Great, thank you for the clarification!
